assets/js/googlesitekit/modules/datastore/settings.js

```diff
@@ -25,7 +25,12 @@ import invariant from 'invariant';
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-const { createRegistrySelector } = Data;
+// import { dispatch } from '@wordpress/data';
+const { createRegistrySelector, createRegistryControl } = Data;
+// import { STORE_NAME } from './constants';
+import { STORE_NAME as CORE_MODULES } from './constants';
```

Comment:
Since this file/module is in the context of the `core/modules` store, `STORE_NAME` should be used to reference its own store rather than the alias.

Reply:
This comment is yet to be addressed.